src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -875,8 +875,11 @@ export default {
             params: { correctedFilingId: item.filingId } })
           break
         case FilingTypes.INCORPORATION_APPLICATION:
-          // for now, not allowed
-          alert('At this time, you cannot correct an incorporation application. Please contact Ops if needed.')
```

Comment:
Why arent we using our standard dialog here?

Reply:
This is defensive coding only so it uses the easy way instead of using the confirm dialog or some custom dialog.

This is existing code. Do you think I should update it?

Reply:
No. This is fine.